tests/end2end/features/yankpaste.feature

```diff
@@ -20,6 +20,19 @@ Feature: Yanking and pasting.
         Then the message "Yanked URL to primary selection: http://localhost:(port)/data/title.html" should be shown
         And the primary selection should contain "http://localhost:(port)/data/title.html"
 
+    Scenario: Yanking URLs with ref and UTM data to clipboard
+        When I open http://kikolani.com/blog-post-promotion-ultimate-guide?utm_source=kikolani&utm_medium=320banner&utm_campaign=bpp
```

Comment:
Also, with that URL, your code could accidentally remove *all* parameters and the test would pass - so you should add some parameters which are not removed.